spec/diddocuments.md

```diff
@@ -226,12 +218,12 @@ For example, a KERI AID with only the following inception event in its KEL:
 ```
 
 #### Verification Relationships
-KERI AID public keys can be used to sign a variety of data.  This includes but is not limited to logging into a website, challenge-response exchanges and credential issuances.  It follows that for each public key in `k` two verification relationships must be generated in the DID document.  One verification relationship of type `authentication` and one verification relationship of type `assertionMethod`.  The `authentication` verification relationship defines that the DID controller can authenticate using each key and the `assertionMethod` verification relationship defines that the DID controller can express claims with each key (should we address multisig and thresholds here?).
+KERI AID public keys can be used to sign a variety of data.  This includes but is not limited to logging into a website, challenge-response exchanges and credential issuances.  It follows that for each public key in `k` two verification relationships must be generated in the DID document.  One verification relationship of type `authentication` and one verification relationship of type `assertionMethod`.  The `authentication` verification relationship defines that the DID controller can authenticate using each key and the `assertionMethod` verification relationship defines that the DID controller can express claims with each key (TODO: address multisig and thresholds).
```

Comment:
Is this a statement about KERI keys, or about DIDs?  I think this is about DIDs in general.  Not sure why it is called out here in this way, as if it was a unique feature of KERI.  The rest of this reads (to me, at least) about normal DIDDoc contents.

Reply:
I think the point here is that KERI AID keys don't generally distinguish between authenticating the controller of the identifier and signing for that controller. In other words, the text is trying to make the point that the same key shows up in two verification relationships (usually), which is not a generally true assumption in all of DID-land.

Reply:
@swcurran although I only meant to specify the TODO with my change, I read the meaning of this section the way that @dhh1128 explained. Since AIDs are KEL backed the key state applies to both verification relationships.

Reply:
Hmmm…still doesn’t seem like something needed to be called out for did:webs.  To create/evolve a the DID, I execute KERI events to accomplish my goal. If that requires I use some keys for authentication to make the changes, I don’t think it need be call out in the DIDDoc section.

Happy to leave it for now (especially since it is merged :-) ), but lets see how it reads in a final review.